docs/integration-association.md

```diff
@@ -0,0 +1,37 @@
+- TO DISCUSS : "members: Set<User>", list of all approved members of the association
+- "e-mail : String", a reference to the e-mail of the association if users want to contact them
+- "website: String? = "" ", a reference to the website of the association for those who have one
+- "instagram: String? = "" ", a reference to the instagram of the association for those who have one
+### AssociationsScreen
+In the hamburger menu, we can add a route to go to an AssociationsScreen where we can find everything related to associations. 
+The first thing a user see is the recent actuality of each associations she/he follows, like a feed (For simplicity, instead of Actuality, we can firstly just show all events created by the association). 
+On top of this page, we have two buttons : one with "Followed Associations" which leads to a list of followed associations and one with "My Associations" which leads to a list of associations in which the user belong. 
+At the bottom right, there is a search button (like the one in HomeScreen) which allows the user to search an association. 
+### FollowedAssociationsPage
+Already designed by Yoan (just needs little update), it consists of the list of all followed associations under the form of a list. Each item of the list have an unsubscribe button. If an item of the list is clicked, it leads the user to the specific AssociationPage.
+### ComiteeAssociationsPage
+Already designed by Yoan (just needs little update), it consists of the list of all associations in which the user belong. If an item of the list is clicked, it leads the user to the specific AssociationPage.
+### SearchMenuAssociations
+When the search button is clicked, it opens a bottom sheet with exactly the same functionnality as Discover, but without the filters (which doesn't make sense to search associations). A list with all associations is displayed in the background and updated according to the bottom sheet. If an item of the list is clicked, it leads the user to the specific AssociationPage.
+### AssociationPage
+On top, we have the choice of two pages : a "Description" default page and a "Actuality" page.
+#### DescriptionPage
+This page consists of the title and description of the association. On top right, there is a button to follow/unfollow the association. If we scroll down, we can see a link to the association website, then a link to the instagram, and an e-mail to contact if the user is interested in joining the association (or for whatever question). If the user belongs to the moderators, she/he has an additional "Edit" button to modify the displayed information.
+#### ActualityPage
+This page shows the history of all posts of the association (like a feed). If the user belongs to the moderators, she/he can post new actualities in the associtations feed for the followers.(For simplicity, instead of Actuality, we can firstly just show all events created by the association)
```

Comment:
You mean you just want to implement it such that it only shows all the activities done by the association sorted by time for the association for now?

Reply:
Yes, and as we discussed in the meeting, we won't have news for association but only the events created by it